Python/pywarpx/picmi.py

```diff
@@ -56,6 +56,7 @@ def init(self, kw):
                     assert self.charge_state <= element.number, Exception('%s charge state not valid'%self.particle_type)
                     try:
                         element = element.ion[self.charge_state]
+                        self.charge = 'q_e'
                     except ValueError:
```

Comment:
Thanks for updating picmi. This line setting charge should go below in the loop over interactions in initialize_inputs. When it is here, this will overwrite the charge even when field ionization is not turned on.

Reply:
Oh sorry, you are right, I will fix this. Thanks for catching this @dpgrote !